library/kotlin/io/envoyproxy/envoymobile/EngineBuilder.kt

```diff
@@ -62,6 +65,21 @@ open class EngineBuilder(
     return this
   }
 
+  /**
+   * Add a loopback port to emit statsd stats to.
+   * Passing nil disables stats emission via the statsd stat sink.
+   *
+   * Only one of the statsd and gRPC stat sink can be enabled.
+   *
+   * @param port The port to send statsd UDP packets to via loopback
+   *
+   * @return this builder.
+   */
+  fun addStatsdPort(port: Int): EngineBuilder {
```

Comment:
```suggestion
  fun addStatsDPort(port: Int): EngineBuilder {
```

Maybe, just to make the "D" a little more obvious? (I misread the first time.)

Reply:
Done